Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_v3.yml

```diff
@@ -0,0 +1,944 @@
+id: Endace Search Archive Download PCAP v3
```

Comment:
What's the difference between v2 and v3? 
Should v2 be deprecated? 
Also, please add the right release notes to this playbook.

Reply:
That is correct v2 should be deprecated. I think the only thing that got changed was that there was a "sleep" command added within the endace-get-search-status

Reply:
So why add a new playbook? if it is the same? 
Why not add the "sleep" command in the v2? if that's the only change?